spinegeneric/cli/generate_figure.py

```diff
@@ -282,8 +282,8 @@ def _get_flag(name):
         Get the flag of a country from the folder flags.
         :param name Name of the country
         """
-        with importlib.resources.path(spinegeneric.flags, f'{name}.png') as path_flags:
-            return plt.imread(path_flags)
+        with importlib.resources.path(spinegeneric.flags, f'{name}.png') as path_flag:
+            return plt.imread(path_flag.__str__())
```

Comment:
@kousu your change with importlib introduced a bug-- this is my fix to it but maybe you had something else in mind?

Reply:
That's the right fix! The python stdlib understands Pathlib objects but I guess plt doesn't. I'm glad you figured it out.